fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -207,6 +207,8 @@ public enum ErrorCode {
             "Unknown resource group '%s' "),
     ERR_BAD_CATALOG_AND_DB_ERROR(5080, new byte[] {'4', '2', '0', '0', '0'},
             "Unknown catalog.db '%s'"),
+    ERR_CATALOG_EXISTED_ERROR(5607, new byte[] {'4', '2', '0', '0', '0'},
```

Comment:
could you give me the error code refrence?

Reply:
Actually, it's no reference for this error code. I just created a new error code for it, which may be a bit unreasonable. Should I remove it and just throw an simple exception?